usaspending_api/awards/models.py

```diff
@@ -310,6 +328,24 @@ def get_default_fields(path=None):
             "assistance_data"  # must match related_name in TransactionAssistance
         ]
 
+    @classmethod
+    def get_or_create(cls, **kwargs):
```

Comment:
Good question, but it doesn't override, because the usual `get_or_create` is defined on `Transaction.objects`, not on `Transaction`.  Note the difference:
```
In [4]: Transaction.objects.get_or_create
Out[4]: <bound method BaseManager._get_queryset_methods.<locals>.create_method.<locals>.manager_method of <django.db.models.manager.Manager object at 0x1044a84e0>>

In [5]: Transaction.get_or_create
Out[5]: <bound method Transaction.get_or_create of <class 'usaspending_api.awards.models.Transaction'>>
```
I chose the name because that's a customary name for getting or creating, but maybe I should change the name to avoid confusion?